projects/core/src/occ/adapters/cms/occ-cms-component.adapter.ts

```diff
@@ -32,11 +36,23 @@ export class OccCmsComponentAdapter implements CmsComponentAdapter {
     id: string,
     pageContext: PageContext
   ): Observable<T> {
-    return this.http
-      .get<T>(this.getComponentEndPoint(id, pageContext), {
+    if (this.featureConfigService.isEnabled('newCmsEndpoint')) {
```

Comment:
Shouldn't we first check if `this.featureConfigService` is not `undefined`?
I was rather expecting:
```
this.featureConfigService?.isEnabled(...)
```

Reply:
it's injected at the top, it will not be undefined